sql/wsrep_utils.cc

```diff
@@ -457,7 +457,13 @@ unsigned int wsrep_check_ip (const char* const addr, bool *is_ipv6)
 
   *is_ipv6= false;
 
-  int gai_ret= getaddrinfo(addr, NULL, &hints, &res);
+  char *end;
+  char address[FN_REFLEN];
```

Comment:
FN_REFLEN was for path lengths so its quite overkill for an address length. `INET6_ADDRSTRLEN` is a good replacement.